app/experimenter/nimbus-ui/src/lib/pluralize.ts

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export default function pluralize(
+  count: number,
+  singular: string,
+  plural?: string,
+) {
+  const pluralVal = plural ?? `${singular}s`;
```

Comment:
This kind of makes my l10n-spidey-sense tingle, but we're not localizing here so 🤷‍♂️